packages/api-file-manager/src/modelModifier/CmsModelModifier.ts

```diff
@@ -14,8 +14,11 @@ class CmsModelFieldsModifier {
     }
 
     addField(field: CmsModelField) {
+        const { bulkEdit, tags, ...rest } = field;
+
         this.fields.push({
-            ...field,
+            ...rest,
+            tags: (tags ?? []).concat(bulkEdit ? ["field:bulk-edit"] : []),
```

Comment:
Not sure about the `field:` prefix. These tags already are "field tags". Maybe let's rename `field:` to `internal:` or `system:`, to designate them as something Webiny handles, and it hsouldn't be changed.

Or, simply adding a `$` prefix, like this `$bulk-edit`? `$` usually means something is a system value.

Reply:
Thanks for the suggestion. I opted for `$bulk-edit`